tasks/managed/add-fbc-contribution/README.md

```diff
@@ -4,16 +4,19 @@ Task to create an internalrequest to add fbc contributions to index images
-| dataPath        | Path to the JSON string of the merged data to use in the data workspace                   | No       | -                    |
-| fromIndex       | fromIndex value updated by update-ocp-tag task                                            | No       | -                    |
-| pipelineRunUid  | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       | -                    |
-| targetIndex     | targetIndex value updated by update-ocp-tag task                                          | No       | -                    |
-| resultsDirPath  | Path to results directory in the data workspace                                           | No       | -                    |
-| taskGitUrl      | The url to the git repo where the release-service-catalog tasks to be used are stored     | No       | -                    |
-| taskGitRevision | The revision in the taskGitUrl repo to be used                                            | No       | -                    |
+| Name                              | Description                                                                               | Optional                | Default value |
+|-----------------------------------|-------------------------------------------------------------------------------------------|-------------------------|---------------|
+| snapshotPath                      | Path to the JSON string of the mapped Snapshot spec in the data workspace                 | No                      | -             |
+| dataPath                          | Path to the JSON string of the merged data to use in the data workspace                   | No                      | -             |
+| fromIndex                         | fromIndex value updated by update-ocp-tag task                                            | No                      | -             |
+| pipelineRunUid                    | The uid of the current pipelineRun. Used as a label value when creating internal requests | No                      | -             |
+| targetIndex                       | targetIndex value updated by update-ocp-tag task                                          | No                      | -             |
+| resultsDirPath                    | Path to results directory in the data workspace                                           | No                      | -             |
+| taskGitUrl                        | The url to the git repo where the release-service-catalog tasks to be used are stored     | No                      | -             |
+| taskGitRevision                   | The revision in the taskGitUrl repo to be used                                            | No                      | -             |
+
+## Changes in 4.0.2
+* Added the `internalRequestServiceAccountName` data parameter, to set which SA should be used to run the internalRequest.
```

Comment:
I don't think calling it a parameter is too good, maybe more like "pass to the internalRequest" also this isn't the proper name - `internalRequestServiceAccount` vs `internalRequestServiceAccountName`

Reply:
Sure. Let me change it. I am happy at least that the e2e finally has passed.

Reply:
fixed.